src/database/queries/addAns.js

```diff
@@ -0,0 +1,5 @@
+const connection = require('../connection.js');
+
+const addAns = (answer, consultent_id)=> connection.query(`INSERT INTO posts (answer, posts_id, consultent_id) values ($1, $2, $3) where posts_id = id returning *`,[answer, posts_id, consultent_id]);
```

Comment:
@denaHS I think you need to add `posts_id` as an argument to this function, i.e. `const addAns = (answer, posts_id, consultent_id) => ...`